### PR TITLE
RIA-7394, RIA-6896, RIA-7393, RIA-7378, RIA-7402, RIA-7379 Fix bunch of issues

### PR DESCRIPTION
**Pull request by AlexanderChernyaev**
https://smartbear.atlassian.net/browse/RIA-7394
https://smartbear.atlassian.net/browse/RIA-7393
https://smartbear.atlassian.net/browse/RIA-7378
https://smartbear.atlassian.net/browse/RIA-7402
https://smartbear.atlassian.net/browse/RIA-7379

### CODE REVIEW

File: src/main/java/com/smartbear/plugins/swaggerhub/ImportFromHubDialog.java [hunk 1]
```diff
@@ -209,7 +226,7 @@ private void populateList() {
 
             SwaggerImporter importer;
 
-            if (descriptor.oasVersion.equals("3.0.0")) {
+            if (descriptor.oasVersion != null && descriptor.oasVersion.equals("3.0.0")) {
```
Comment (antone-sb) [inline src/main/java/com/smartbear/plugins/swaggerhub/ImportFromHubDialog.java]: Use StringUtils.sameString().

File: src/main/java/com/smartbear/plugins/swaggerhub/ImportFromHubDialog.java [hunk 2]
```diff
@@ -47,17 +48,22 @@
...
 import org.json.JSONObject;
 
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
 import static com.smartbear.swagger.AbstractSwaggerImporter.AUTHORIZATION_HEADER;
+import static javafx.scene.control.Alert.AlertType.WARNING;
```
Comment (antone-sb) [inline src/main/java/com/smartbear/plugins/swaggerhub/ImportFromHubDialog.java]: Use this for all types or import javafx.scene.control.Alert.AlertType only.

**Review by antone-sb — CHANGES_REQUESTED** [review]

File: src/main/java/com/smartbear/plugins/swaggerhub/ImportFromHubDialog.java (hunk 2 above)
Comment (AlexanderChernyaev) [reply src/main/java/com/smartbear/plugins/swaggerhub/ImportFromHubDialog.java]: Fixed

File: src/main/java/com/smartbear/plugins/swaggerhub/ImportFromHubDialog.java (hunk 1 above)
Comment (AlexanderChernyaev) [reply src/main/java/com/smartbear/plugins/swaggerhub/ImportFromHubDialog.java]: Fixed

**Review by antone-sb — APPROVED** [review]

**Pull request merged**